<launch>    
  <node pkg="multiview_object_recognizer" type="multiview_object_recognizer_node" name="multiview_object_recognizer_node" output="screen" >
        <param name="models_dir" value="/media/Data/datasets/TUW/models/"/>
        <param name="training_dir_sift" value="/media/Data/datasets/TUW/sift_trained/"/>
        <param name="training_dir_shot" value="/media/Data/datasets/TUW/shot_trained/"/>
        <param name="recognizer_structure_sift" value="/media/Data/datasets/TUW/training_data/"/>
        <param name="visualize_output" value="false"/>
        <param name="scene_to_scene" value="true"/>
        <param name="do_sift" value="true"/>
        <param name="do_ourcvfh" value="false"/>
        <param name="do_shot" value="false"/>
        <param name="chop_z" value="2.0"/>
        <param name="cg_size_thresh" value="7"/>
        <param name="cg_size" value="0.015"/>
        <param name="cg_ransac_threshold" value="0.015"/>
        <param name="cg_dist_for_clutter_factor" value="0"/>
        <param name="cg_max_taken" value="2"/>
        <param name="cg_max_time_for_cliques_computation" value="100"/>
        <param name="cg_dot_distance" value="0.2"/>
        <param name="hv_resolution" value="0.005"/>
        <param name="hv_inlier_threshold" value="0.015"/>
        <param name="hv_radius_clutter" value="0.03"/>
        <param name="hv_regularizer" value="3"/>
        <param name="hv_clutter_regularizer" value="5"/>
        <param name="hv_occlusion_threshold" value="0.01"/>
        <param name="hv_optimizer_type" value="0"/>
        <param name="hv_color_sigma_l" value="0.6"/>
        <param name="hv_color_sigma_ab" value="0.5"/>
        <param name="icp_iterations" value="10"/>
        <param name="icp_type" value="1"/>
        <param name="icp_voxel_size" value="0.005"/>
        <param name="max_vertices_in_graph" value="10"/>
        <param name="distance_keypoints_get_discarded" value="0.000025"/>
    </node>
</launch>
